spec/guard/teaspoon/runner_spec.rb

```diff
@@ -68,6 +61,18 @@
       subject.run([], baz: "teaspoon")
     end
 
+    it 'shows filenames when the show_filenames flag is set' do
+      files = ["file1", "file2"]
+      expect(Guard::Compat::UI).to receive(:info).with("Running files: file1, file2")
+      subject.run(files, show_filenames: true)
+    end
+
+    it 'does not show filenames when the show_filenames flag is set to false' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.